financial-templates-lib/test/ExpiringMultiPartyClient.js

```diff
@@ -310,6 +310,11 @@ contract("ExpiringMultiPartyClient.js", function(accounts) {
       ],
       expiredLiquidations
     );
+
+    // Withdraw from the expired liquidation and check that the liquidation is deleted.
+    await emp.withdrawLiquidation("0", accounts[0], { from: accounts[1] })
```

Comment:
I know not directly related to your PR but we should probs not use `accounts[0]` and `accounts[1]` in the tests like this to be consistant.

Reply:
renamed.